src/pgrn-custom-scan.c

```diff
@@ -781,6 +781,8 @@ PGrnMakeTupleTableSlot(CustomScanState *customScanState,
 	ListCell *cell;
 	ExecClearTuple(slot);
 	econtext->ecxt_scantuple = slot;
+	Snapshot snapshot = SnapshotAny;
+	snapshot = RegisterSnapshot(GetTransactionSnapshot());
```

Comment:
Can we simplify them?

```suggestion
	Snapshot snapshot = RegisterSnapshot(GetTransactionSnapshot());
```